components/Wallet/Send.js/GasCustomization.jsx

```diff
@@ -171,6 +171,10 @@ const GasCustomization = ({
       </Box>
       <FloatingContainer>
         <Button
+          border={0}
```

Comment:
@as-dr why are these borders being put on the button itself? Are we veering away from theme here?

Reply:
I think the FloatingContainer is poorly worded and it's more of a StepperBar or something that lets you move back/forward through a flow. 

I don't think we're deviating from the theme massively, because the core styling of the primary and secondary buttons inside this component remain the same. It's just their shape/positioning within the context of the FloatingContainer that are changing - to look cleaner.

I think it's acceptable, but always happy to entertain alternatives. 😇

Reply:
Fair enough